### ISSUE TITLE
Bug: Game sound doesn't play on opening the game on web browser

### ISSUE THREAD

**Issue by superiorsd10** [flutter/super_dash#179]
### Description

When you open the game in any web browser, the game sound doesn't play. On muting, and unmuting the game sound through the sound button, it starts playing. 

I'm using the Arc browser. 

To emulate this bug, just open this game on any web browser.

### Screen Recording

https://github.com/flutter/super_dash/assets/92971894/861c9967-7c7e-4f77-a336-9c58a4b4f66b

_I would love to work on this issue and contribute to this project, so please consider assigning it to me._ 

_Thank you,_

**Comment by ditman** [comment]
(PS: @superiorsd10 if you want to give a shot to the fix, go for it!)

**Comment by superiorsd10** [comment]
@ditman Yes, I would love to give it a go and make a PR if I can find any solution.
